src/include/btmem.h

```diff
@@ -1187,17 +1186,17 @@ struct __wt_update_value {
  * avoid heap allocation, add a few additional slots to that array.
  */
 #define WT_MAX_MODIFY_UPDATE 10
-#define WT_MODIFY_VECTOR_STACK_SIZE (WT_MAX_MODIFY_UPDATE + 10)
+#define WT_UPDATE_VECTOR_STACK_SIZE 20
```

Comment:
Curious why we are increasing the stack size?

Reply:
It's the same size. I just don't want the stack size to depend on the number of modies.